### ISSUE TITLE
⚠️ BUILTINS : OUTPUT STREAM MISFUNCTIONS

### ISSUE THREAD

**Issue by OussamaBOUIZ** [ibranad/Minishell_42#17]
`export > file` this commands writes to STDINPUT rather than writing to the given file
`echo "some text" > file` this as well falls into the same problem.
This concerns just BUILTINS

**Comment by OussamaBOUIZ** [comment]
## Detections:
- The given file doesn't get created

**Comment by OussamaBOUIZ** [comment]
**This issue is true fixed in 2 steps:**
1. When running a BUILTIN the app wasn't allocating for it the right output stream. So we had to add this piece of code to make it at least the outputting correct `write_to(cmd->out_fd)` .
2. Then it turns out that the application outputs to the right output stream but the program hangs there, the problem is when we specify for it the output stream and we are not doing it inside a child process since we running a builtin command we're reshaping the output stream and hence we lose **STDOUT**, the fix of this is reachable by saving the STDOUT before get in to the loop  the same thing we did in #6 but this time with **STDOUT**
